synapse/config/metrics.py

```diff
@@ -97,4 +98,8 @@ def generate_config_section(self, report_stats=None, **kwargs):
         else:
             res += "report_stats: %s\n" % ("true" if report_stats else "false")
 
+        res += """\
+        # The endpoint to report the anonymized homeserver usage statistics to.
+        #report_stats_endpoint: https://example.com/report-usage-stats/push
```

Comment:
This doesn't follow the style in coding_style.rst

```suggestion
        #
        #report_stats_endpoint: https://example.com/report-usage-stats/push
```

also please can you document the default